modules/currency-l0/src/main/scala/io/constellationnetwork/currency/l0/modules/HttpApi.scala

```diff
@@ -43,7 +45,8 @@ object HttpApi {
     nodeVersion: TessellationVersion,
     httpCfg: HttpConfig,
     maybeDataApplication: Option[BaseDataApplicationL0Service[F]],
-    maybeMetagraphVersion: Option[MetagraphVersion]
+    maybeMetagraphVersion: Option[MetagraphVersion],
+    currencySnapshotValidationErrorStorage: ValidationErrorStorage[F, CurrencySnapshotEvent, Option[BlockRejectionReason]]
```

Comment:
Technically it doesn't store snapshot validation errors. These are errors of snapshot events so inputs for creating the currency snapshot.